### ISSUE TITLE
Build fails on fedora-32-armhfp

### ISSUE THREAD

**Issue by hegjon** [cculianu/Fulcrum#62]
```
gcc -c -pipe -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -march=armv7-a -mfpu=vfpv3-d16 -mtune=generic-armv7-a -mabi=aapcs-linux -mfloat-abi=hard -DNDEBUG -fomit-frame-pointer -O2 -w -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DUSE_QT_IN_BITCOIN -DHAVE_ENDIAN_H -DHAVE_DECL_HTOBE16 -DHAVE_DECL_HTOLE16 -DHAVE_DECL_BE16TOH -DHAVE_DECL_LE16TOH -DHAVE_DECL_HTOBE32 -DHAVE_DECL_HTOLE32 -DHAVE_DECL_BE32TOH -DHAVE_DECL_LE32TOH -DHAVE_DECL_HTOBE64 -DHAVE_DECL_HTOLE64 -DHAVE_DECL_BE64TOH -DHAVE_DECL_LE64TOH -DHAVE_JEMALLOC_HEADERS -DQT_NO_DEBUG -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Isrc -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtNetwork -isystem /usr/include/qt5/QtCore -I. -I/usr/lib/qt5/mkspecs/linux-g++ -o secp256k1.o src/bitcoin/secp256k1/secp256k1.c
In file included from src/bitcoin/secp256k1/secp256k1.c:18:
src/bitcoin/secp256k1/util.h:110:37: error: '__int128' is not supported on this target
  110 | SECP256K1_GNUC_EXT typedef unsigned __int128 uint128_t;
      |                                     ^~~~~~~~
```

See https://download.copr.fedorainfracloud.org/results/jonny/BitcoinCash/fedora-32-armhfp/01706384-fulcrum/builder-live.log.gz for details

**Comment by cculianu** [comment]
Hmm yeah I have to figure out how to fix that -- I noticed the same thing on my RPi.  The funny thing is we don't even use secp in Fulcrum -- I threw it in in case later I wanted to add crypto  signing/verifying features to Fulcrum.  It's entirely superfluous.

I remember trying to build on ARM-32 and hacking together a build that doesn't even include libsecp256k1.  It built but then it had bugs when running.

So -- even if this were fixed it still would need work.  I'm leaving this issue open.  Thanks for reporting.
